flocker/control/schema/types.yml

```diff
@@ -1,16 +1,19 @@
 $schema: http://json-schema.org/draft-04/schema#
 id: http://api.clusterhq.com/v1/types.json
 definitions:
+  uuid:
```

Comment:
Dang and blast it, the `$ref` to this definition in the types that use it mean that their titles and descriptions are ignored. Some sort of solution using `allOf` is the only thing I can think of, but I don't think our docs builder for the API reference can handle that. I'll look at the options in more detail - obviously the simplest solution is to use a generic title and description in the type definition here, but that would probably make some sections of the API reference confusing, because we use UUIDs for both nodes and datasets.